tests/test_browser.py

```diff
@@ -5010,6 +5010,10 @@ def test_emscripten_performance_now(self):
   def test_embind_with_pthreads(self):
     self.btest('embind_with_pthreads.cpp', '1', args=['--bind', '-s', 'USE_PTHREADS', '-s', 'PROXY_TO_PTHREAD'])
```

Comment:
I wonder if we can/should just extend the existing `embind_with_pthreads.cpp` here?

Reply:
I wanted to avoid changing existing tests (although I heavily borrowed from this test); 
The thing I am testing with the new case is whether multiple threads can call some bound function whereas this test seems to just be doing some sanity tests for embind with pthread

Reply:
Sure, but `embind_with_pthreads.cpp` also does exactly that, it just does it from only one thread (despite its name).  

I think extending `embind_with_pthreads.cpp` is fine and probably good idea rather than trying to create an entirely new test with a rather convoluted name.  I think calling functions is kind of the bare minimum to consider embind working at all with threads isn't it?   WDYT @RReverser ?

Also what aren't these tests in tests/embind like the rest of the embind tests?  (we can address that separately but I'm just curious if there is a reason?)

Reply:
Sounds good, I'll merge this into a single test if that option is preferable. Re why they aren't in `embind/`, I'm not entirely sure, I'll just move the test inside embind directory if you don't mind